Light.json

```diff
@@ -43597,7 +43609,7 @@
         "icons": [
           "Episode I"
         ],
-        "gametext": "Use 5 Force to reveal opponent's hand. If opponent has no duplicate cards there, place this Interrupt in Used Pile. Otherwise USED: place a card revealed in opponent's Used Pile,LOST: all cards opponent has two or more of are lost. (Immune to Sense.)",
+        "gametext": "Use 5 Force to reveal opponent's hand. If opponent has no duplicate cards there, place this Interrupt in Used Pile. OtherwiseUSED: place a card revealed in opponent's Used Pile,LOST: all cards opponent has two or more of are lost. (Immune to Sense.)",
```

Comment:
We want to keep the space after "Otherwise" right?  And I think a period after "Used Pile" and before " LOST:"

Reply:
Yeah.  This is one of the only cards that has inline USED/LOST text like this.  I'll have to update my CDF-creating parser to account for this edge case (instead of making the text look bad like I did to appease the parser).

I will put the space back in before USED.  Card image shows a comma before LOST.